config/jobs/cert-manager/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -9,7 +9,7 @@ presubmits:
     decorate: true
     branches:
     - master
-    - release-1.8
+    - release-1.9
```

Comment:
Suggestion: as I mentioned in the 1.8.0 release thread on slack, I didn't create `release-1.9`. We can just test master here

Reply:
I agree, let’s get rid of release-1.9 for now until we do an alpha release